### PR TITLE
Fixes n tweaks

### PR DESCRIPTION
**Pull request by Freddo3000**
* Changed ACRE signal mode from multi-path to simple LOS (Multi-path seems to be about as unstable and unreliable as in real life 🤔)
* Reduced AI hearing distance (Should hopefully get less people complaining about them seeing through bushes)
* Fixed medical items being in USM backpack by default (@AChesheireCat) 
* Disable CSW ammo handling for AI, should now be able to properly use statics
* Force hide vanilla parachute altimeter. Disabled by default, but now it is forced.
* G-Forces enabled at 50% strength. Previously disabled due to not being able to tweak the strength, either being off or on.
* Increased suppression intensity from 0.3 > 0.4
* Enable changing insignia and face in arsenal, and sharing loadouts. Don't see a reason to have it disabled.
* Enable JIP at all times (@Feebee21)

### CODE REVIEW

**Review by BorderKeeper — APPROVED** [review]
agreed with above. Only thing I am worried is messing with ACRE again, but looking at the issues I have been experiencing lately with signal loss this is the best choice.

**Comment by RamenNoodleNinja** [comment]
ACRE change is likely to get more complaints, not fewer. LOS Simple means any obstacle for direct LOS means degraded or nonexistant comms. ACRE pushed a new version as well, we probably want to wait and see what changes first.

What's the reason for upping suppression?

Why force hide the altimeter? Only the people who want to see it see it, and as one of those people I prefer to keep it. No one ever gives us the altimeter watches anyway.

**Comment by Freddo3000** [comment]
> 
> 
> ACRE change is likely to get more complaints, not fewer. LOS Simple means any obstacle for direct LOS means degraded or nonexistant comms. ACRE pushed a new version as well, we probably want to wait and see what changes first.

Currently there are seemingly random circumstances where radios will cease to function whatsoever. By simplifying the simulation I'm hoping to resolve that, I've also reduced the terrain coefficient to compensate.

I'll revert the changes for now, seeing that there was a update that dropped just now.

> 
> What's the reason for upping suppression?

It was something that was discussed when we were introducing it to start at low numbers and then progressively increase it as time goes on. To be clear I appreciate feedback on this, which is why I link this stuff before pushing it.
> 
> Why force hide the altimeter? Only the people who want to see it see it, and as one of those people I prefer to keep it. No one ever gives us the altimeter watches anyway.

Well the reason for it is to require the altimeter watches to simply put, otherwise I'd enable it globally to not give people an unfair advantage, even if it is the tiniest thing ever.

**Comment by RamenNoodleNinja** [comment]
Don't think player advantage really comes into it a whole lot, since I don't think we've ever paradropped in a PvP, and are unlikely to hot drop directly onto AI for reasons other than our parachute opening height being "optimal".

I'm all for making altimeter watches a norm if we have a paradrop, but the default is off, so why is this something that needs to be forced? If we're going to have altimeter watches, we'll have the info anyway, and the vanilla UI might be preferred over the ACE watch.

I think as a general rule, if the thing to be forced has a low chance for abuse, we should leave it to players' choice unless there is actual abuse. The exception being where lack of uniformity would cause clear confusion. E.g if the STUI choice of colors was down to player choice, so there was no guarantee that Blue was a color choice for all members, or if the concenctric rings were variable sizes per person and the lead element asks for a spacing of 2 rings.

**Comment by Freddo3000** [comment]
To be honest I don't care in regards to the paradrop HUD thing, it is you say such a minor thing. The reason I forced it was for a tiny bit more consistency. I can leave it either way.

On the suppression stuff, @BorderKeeper signed off on it by approving it, and he was present when we were testing it IIRC. Correct me if I'm wrong.

**Comment by BorderKeeper** [comment]
I always felt that the default suppression was low from original values. I would still have an admin give ok for this merge to happen. ACRE we can leave behind for now and I agree with @RamenNoodleNinja on the altimeter thing.

**Comment by RamenNoodleNinja** [comment]
If we're not forcing the suppression stuff, just changing the default, I say go for it.

**Comment by Freddo3000** [comment]
> 
> 
> If we're not forcing the suppression stuff, just changing the default, I say go for it.

It is forced as otherwise people would simply be able to outright turn it off.

**Comment by RamenNoodleNinja** [comment]
So I just went through a bunch of Discord and the comments on the pull requests, and the only thing I saw regarding suppression was Church's comment way back in July about being concerned when the buildup was changed from 1.0 to 1.3. No mention at all of slowly ramping up suppression. I don't want it to be forced, especially not if it's higher than now. Someone turning it off completely is likely to be so rare that we can deal with it per-person, and turning it up for everyone with no way for it to be turned down/off isn't really a good solution/compromise with "I like where the setting is", "I want it higher", and "I will secretly turn this off". Two of those haven't really been discussed in a meaningful way as far as I can tell, and the third is a different issue than a forced setting can address.

**Comment by Freddo3000** [comment]
This was discussed on TS in multiplayer while we were testing for values. Nothing on discord or on here unfortunately. At least @BorderKeeper and @hazardouschurch were present while picking the initial values, though there were a few others as well although this was several months back.

Whatever value we'll pick must be forced, as we cannot change the defaults nor the ranges for the settings. Being allowed to turn it off will also throw a major wrench into PVP balance, as the default values when not forced are extreme in comparison to what we have currently.

**Comment by hazardouschurch** [comment]
I think we should leave the suppression where it was. In my mind its more of a tool to learn when people are getting shot. We have never had the intention of forcing visual mods. The one exception to this is the smoke mod which was seen a fix to lackluster smoke for pilots. The acre change may be a mute point as it will get updated with the big update.

**Pull request merged**
